RecoEgamma/EgammaPhotonProducers/python/egammaForCoreTracking_cff.py

```diff
@@ -0,0 +1,58 @@
+import FWCore.ParameterSet.Config as cms
+
+
+
+from RecoEcal.EgammaClusterProducers.particleFlowSuperClusterECAL_cfi import particleFlowSuperClusterECAL as _particleFlowSuperClusterECAL
+
+#cant use the regression as requires #vertices and we cant use tracking info
+#also require it to be above 20 GeV as we only want E/gammas Et>20 and H/E <0.2
+particleFlowSuperClusterECALForTrk = _particleFlowSuperClusterECAL.clone(
+     useRegression = cms.bool(False),
+     regressionConfig = cms.PSet(),
+     thresh_SCEt = cms.double(20.0)
+)
+
+egammasForTrk = cms.EDProducer( "EgammaHLTRecoEcalCandidateProducers",
+    scIslandEndcapProducer = cms.InputTag( 'particleFlowSuperClusterECALForTrk','particleFlowSuperClusterECALEndcapWithPreshower' ),
+    scHybridBarrelProducer = cms.InputTag( 'particleFlowSuperClusterECALForTrk','particleFlowSuperClusterECALBarrel' ),
+    recoEcalCandidateCollection = cms.string( "" )
+)
+
+egammaHoverEForTrk = cms.EDProducer( "EgammaHLTBcHcalIsolationProducersRegional",
+    effectiveAreas = cms.vdouble( 0.105, 0.17 ),
+    doRhoCorrection = cms.bool( False ),
```

Comment:
shouldn't this be corrected?

Reply:
Yes. Although I havnt figured out how to best do that yet. And its mimicking the H/E cut on ecal driven electrons which isnt corrected (and really should be...). On this list for the H/E revamp.